app/javascript/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Routes, Route, Link } from 'react-router-dom';
+import Greeting from './components/Greeting';
+
+const App = () => {
+  return (
+    <>
+      <h1>Hello World!</h1>
+      <Link to="/greeting">Go to Greetings Page</Link>
+      <Routes>
+        <Route path="/greeting" element={<Greeting />} />
+      </Routes>
+    </>
+  );
+};
+
+export default App;
```

Comment:
- [ ] The API returns the message, but I cannot see this in UI in this (**/greeting**) URL.

Reply:
That is "/api/greeting" bro. Not "/greeting". But when you click the link you can see the message. Want me to show you demo? @Deepakdanger